commands/run_test.go

```diff
@@ -165,11 +170,11 @@ func TestDefineResources(t *testing.T) {
 		expectedErrorMsg string
 	}{
 		{"../examples/addition-job/seed.manifest.json",
-			4.0, "[-m 16m --shm-size=128m]", 5.0, true, ""},
+			4.0, "[-e CPU=0.100000 -m 16m -e MEM=16 -e DISK=5.000000 --shm-size=128m -e SHAREDMEM=128]", 5.0, true, ""},
 		{"../examples/extractor/seed.manifest.json",
-			1.0, "[-m 16m --shm-size=1m]", 1.01, true, ""},
+			1.0, "[-e CPU=10.000000 -m 16m -e MEM=16 --shm-size=1m -e SHAREDMEM=1 -e DISK=1.010000]", 1.01, true, ""},
```

Comment:
I'd bring these examples so they don't have to consume 10 cores... probably don't want more than 4.